testdata/picard/illumina/mark_illumina_adapters_test.sam

```diff
@@ -1,4 +1,4 @@
 @HD	VN:1.6	SO:queryname
-@RG	ID:62A79.3	PL:ILLUMINA	PU:62A79AAXX100907.3	LB:Solexa-41734	DT:2010-09-07T00:00:00-0400	SM:NA12878	CN:BI
+@RG	ID:62A79.3	PL:ILLUMINA	PU:62A79AAXX100907.3	LB:Solexa-41734	DT:2010-09-06T22:00:00-0600	SM:NA12878	CN:BI
```

Comment:
What happened here?

Reply:
weird! I don't see this timestamp anywhere else in the codebase. I'll switch it back, but it is freaky